contracts/vaults/base/RibbonVault.sol

```diff
@@ -18,12 +26,52 @@ import {
     IStrikeSelection,
     IOptionsPremiumPricer
 } from "../../interfaces/IRibbon.sol";
+import {IRibbonThetaVault} from "../../interfaces/IRibbonThetaVault.sol";
 
-contract RibbonVault is OptionsVaultStorage {
+contract RibbonVault is
+    ReentrancyGuardUpgradeable,
+    OwnableUpgradeable,
+    ERC20Upgradeable
+{
     using SafeERC20 for IERC20;
     using SafeMath for uint256;
     using ShareMath for Vault.DepositReceipt;
 
+    /************************************************
+     *  NON UPGRADEABLE STORAGE
```

Comment:
perhaps we still move the storage variables to a different contract for readability?

Reply:
There isn't really a point of moving it into a separate contract because we don't intend to upgrade it. The inheritance chain also makes it hard to follow